### DISCUSSION TITLE
Resizing of frozen table columns does not work as expected

### DISCUSSION THREAD

**Discussion by ThoSap** [primefaces/community#1477] [PrimeNG] [answered]
In the following StackBlitz example, we've frozen the first two columns `Name` and `Id`.
If you make the `Name` column smaller (resize), the `Country` column will overlap with the not-frozen columns and be visible behind and between the `Name` and `Id` columns.

https://stackblitz.com/edit/wgwshv?file=src%2Fapp%2Fdemo%2Ftable-frozen-columns-demo.html

This was already reported on the 14 November and 21 December 2022 in the following two GitHub issues:
https://github.com/primefaces/primeng/issues/12210
https://github.com/primefaces/primeng/issues/12411

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for reporting the issue, column resize is incompatible with frozen columns. I'll add this issue to the future milestone. We add this support in v17 or v18.

**Reply by lf-novelt** [reply]
hello, is this tracked in a new issue? there don't seem to be any open issue related to this bug which still occurs in latest versions of v17 https://github.com/primefaces/primeng/issues?q=is%3Aissue+frozen+column+resizable

the most recent one https://github.com/primefaces/primeng/issues/12411 was closed in Dec 2024 while not being fixed
